### ISSUE TITLE
JsonUnwrapped prefix ignored in release 2.7.0

### ISSUE THREAD

**Issue by ccdd4ever** [springfox/springfox#2182]
release 2.7.0 fixed the JsonUnwrapped  problem ,but prefix still didn't show.

`
@Data
public class ProductVO {
private String name;
@JsonUnwrapped(prefix = "specification_")
private Specification specification;
}
`

`
@Data
public class Specification {
private String name;
}
`

what i got in swagger-ui is something like

`
ProductVO {
 name(string,optional);
}
`

**Comment by jcarnus** [comment]
For what I've seen in the code, the prefix and suffix of JsonUnwrapped are not implemented in the code. For personal purpose, I need it and implement it here dirty https://github.com/springfox/springfox/pull/2004/commits/29a85493de9fb29c859e86a491739229277fca0c.

Could you let me know how to well coding it and I can have a look ?

**Comment by jcarnus** [comment]
Also, please note that with Jackson, multiple prefix unwrapped are concatenated. The impl must support this

**Comment by jcarnus** [comment]
The impl works as expected but doesn't support multiple JsonUnwrapped nested with a prefix or suffix set. Can works on it if interested!

**Comment by dilipkrish** [comment]
@jcarnus Thats true I did not add support for suffix as that wasn't on the bug request. I did add support for nested unwrapped. See example above ☝️ 

If you have bandwidth would love for support for suffix as well. Again thank you very much for the pointer to your implementation. It made it very simple. I would've waited for your PR, but Im trying to get 2.8.1 out the door soon.
